Twig/Extension/PostMetaExtension.php

```diff
@@ -49,6 +49,7 @@ public function getFunctions()
     {
         return array(
             new \Twig_SimpleFunction('wp_get_post_meta', array($this, 'getPostMeta')),
+            new \Twig_SimpleFunction('wp_get_image_url_from_id', array($this, 'getImageUrlFromId')),
```

Comment:
just checked the Twig code, and it should work: https://github.com/twigphp/Twig/blob/604e94883bc1691071d9a30ff522808dc476394c/test/Twig/Tests/Node/Expression/FunctionTest.php#L31